lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -1322,7 +1322,8 @@ defmodule Phoenix.LiveViewTest do
 
   """
   def assert_patched(view, to) do
-    assert_patch(view, to, 0)
+    {path, _flash} = assert_navigation(view, :patch, to, 0)
```

Comment:
Let's change `assert_patch` to return the path as well, that's more useful than it returning `:ok`. :)